### ISSUE TITLE
[Bug]: Slack Huddle : Can't open the screen sharing preview

### ISSUE THREAD

**Issue by ragusa87** [ferdium/ferdium-recipes#243]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-recipes/blob/develop/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-recipes/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ferdium/ferdium-recipes/issues) for a feature request that matches the one I want to file, without success.

### Ferdium Version

6.2.0

### Recipe Name

slack

### Recipe Version

b14a52d

### Steps to reproduce

1. Add a "slack" service and login
2. Join a channel and start a huddle
3. Share your own screen
4. Click on the screen preview to maximize it or on the "View Shared Screen" button (see screenshot)

### Expected Behavior

The shared screen should be opened in a new window so you can see the screen content

### Actual Behavior

Nothing happen, the new window doesn't open.

### Screenshots

![image](https://user-images.githubusercontent.com/1695207/203286563-d2b27066-24bc-4e6e-bc4c-dd0d7f29574f.png)

### Additional Information

Logs:
```
[42529:1122/105853.551585:ERROR:egl_dmabuf.cc(569)] Failed to record frame: Error creating EGLImage - EGL_NOT_INITIALIZED
[42529:1122/105853.551637:ERROR:shared_screencast_stream.cc(655)] Dropping DMA-BUF modifier: 72057594037927935 and trying to renegotiate stream parameters
(electron) The new-window event is deprecated and will be removed. Please use contents.setWindowOpenHandler() instead.

```

**Comment by ragusa87** [comment]
Slack recipe on commit b14a52d is version 1.4.2

**Comment by kciredor** [comment]
> This issue also occurs when you try to view shared screens from other people in the huddle.

Exactly, which is the reason I wanted to create an issue here, but it has already been filed ;-)

**Comment by jurienhamaker** [comment]
This is turning into quite the hassle for me with my work slack. Would love to see a fix. I'll try check if I can do anything myself.

Also kind of annoying how slack pushes huddles this hard, since not everyone loves it that much, but that's a discussion for somewhere else.

**Comment by jsanta** [comment]
Same problem here. Trying to opt out from Rambox and Slack huddles are the only thing  stopping me from doing so.

**Comment by jdegger** [comment]
Confirming this bug also exists on macOS
